functionary/ui/urls.py

```diff
@@ -166,6 +166,7 @@
         (scheduling.function_parameters),
         name="function-parameters",
     ),
+    path("task_log/", (tasks.get_task_log), name="task-log"),
```

Comment:
this path should probably be `/task/<pk>/log`

Reply:
Fixed. 

I tried moving the `tasks.get_task_log` to the existing `TaskResultsView`, but it already had a get method defined. Maybe it's worth just adding some additional logic to that view's get method to handle the `task/<pk>/log` endpoint? Not too sure if this is the correct approach, or if I should just make a new view

Reply:
Let's leave it as is for now.  I struggled a bit with this same thing in the workflows branch.  It seems inevitable that we're going to end up with a mix of view classes and one-off functions, but I'm sure there will be opportunities to consolidate or just generally more nicely organize things as we go along.